lib/analytics.js

```diff
@@ -36,6 +36,10 @@ Analytics.prototype = {
 
   topNoResultQueries: function(params, callback) {
     this.client.get('/engines/{engine}/analytics/top_no_result_queries.json', params, callback)
+  },
+
+  logClickthrough: function(params, callback) {
+    this.client.post('/engines/{engine}/document_types/{documentType}/analytics/log_clickthrough.json', params, callback);
```

Comment:
The convention in this project is no semi-colons.

Reply:
No problems - will follow the convention